### DISCUSSION TITLE
Adonis future / contingency plans

### DISCUSSION THREAD

**Discussion by etmartinkazoo** [adonisjs/core#3958] [General]
I'm building a pretty large production app in Adonisjs and I love the framework. It is really a pleasure to work in.

My question is, what plans are in place to ensure that the project continues in the future. Contingency plans like what if the founder/maintainer gets sick, moves to another industry of work, passes away, etc... Is there some structure in place to help guide it through such events. I certainly hope nothing of the sort transpires but I was just curious as I would like to build more projects with it for clients but have questions around its long term maintainability/stability.

Again, sorry if this is a less exciting question and I love the project!

**Comment by McSneaky** [comment]
Currently there are some generouse backers for AdonisJS

And Virk isn't working alone on it anymore, there are quite some people in "Core Team", so it won't stop when one maintainer leaves, even if it's going to be Virk (who'se main maintainer)

We have whole teams building products on top of Adonis, went in big with that 😋

**Reply by etmartinkazoo** [reply]
Thank you @McSneaky I really appreciate your response here. It definitely gives me some helpful context and the confidence I need to keep using the framework, which I will!
